runtime/dancebox/src/lib.rs

```diff
@@ -704,13 +726,90 @@ impl pallet_authority_assignment::Config for Runtime {
     type AuthorityId = NimbusId;
 }
 
+pub struct ChargeForBlockCredit<Runtime>(PhantomData<Runtime>);
```

Comment:
We probably can move this inside pallet-service-payment right?

Reply:
Moving this to pallet_services_payment adds a dependency to pallet_balances, is that ok?

Reply:
Nevermind, it does not